### PR TITLE
Config backup

### PR DESCRIPTION
**Pull request by gsanchietti**
See https://github.com/NethServer/ns8-core/pull/344#issuecomment-1443338774

Implements: https://trello.com/c/iSBTafzj/153-core-p1-change-cluster-backup-encryption-password
Fixes: https://trello.com/c/hO89TA6x/318-core-p1-disaster-recovery-not-working

### CODE REVIEW

**Comment by gsanchietti** [comment]
Changes to implement:

- the backup will always be encrypted with the password given by the user
- if the user doesn't enter a password, the backup is *not* encrypted
- the UI will force the user to enter a password: the backup page will not show the application backup until a password has been set
- the password will not be shown inside the UI after has been set
- the restore procedure must always ask for the encryption password
- future: add the cluster backup to the repository, notify the user about it inside the schedule

File: core/ui/public/i18n/en/translation.json [hunk 1]
```diff
@@ -481,19 +482,24 @@
...
-    "download_cluster_configuration_backup": "Download backup",
+    "apps_configuration_tooltip": "Add your backup repositories and configure apps backup scheduling",
+    "restore_tooltip": "You can restore any previously backed up app or the whole cluster:",
+    "backup_page_tooltip_8": "<strong>Diaster recovery</strong>: you will need a cluster configuration backup and a fresh installation of {productName}. The disaster recovery can be achieved only from the cluster configuration wizard.",
+    "download_cluster_configuration_backup": "Download cluster backup",
     "restore_app": "Restore an app",
     "replace_existing_app": "Replace existing app {app}",
     "cluster_configuration": "Cluster configuration",
-    "apps": "Apps",
+    "apps": "Applications",
```
Comment (andre8244) [inline core/ui/public/i18n/en/translation.json:492]:
```suggestion
    "apps": "Apps",
```
We use the term "app" everywhere else, maybe it's better to be consistent

**Review by andre8244 — COMMENTED** [review]
Just a couple of typos

File: core/ui/public/i18n/en/translation.json (hunk 1 above)
Comment (gsanchietti) [reply core/ui/public/i18n/en/translation.json:492]: I really do not like it, but ok ...

**Review by andre8244 — APPROVED** [review]

**Pull request merged**